unittest/sql/open_address_hash-t.cc

```diff
@@ -32,7 +32,7 @@ struct pointer_trait
 
 struct test_value_trait: public pointer_trait<uint32>
 {
-  static const uint32 &get_key(uint32 *elem) { return *elem; }
+  static const uint32* get_key(uint32 *elem) { return elem; }
```

Comment:
1. put `*` sticked to `get_key`: `uint32 *get_key`
2. add `const` to the argument.

Reply:
adding const to an argument causes a compilation error. unable to convert TABLE_LIST

Reply:
there's no TABLE_LIST here, it's a unit test